### PR TITLE
Renders accessible the buttons made from link element 

### PR DESCRIPTION
**Pull request by radinamatic**
Added `role="button"` to links which act like buttons, codebase wide, so they can be perceived as buttons by assistive technologies. Plus some minor a11y fixes.

Should be OK to merge.

### CODE REVIEW

**Comment by radinamatic** [comment]
@aronasorman @rtibbles 

Another with cleared tests and ready to merge!

**Comment by rtibbles** [comment]
I think you may have merge conflicted with yourself on this one :)

**Comment by radinamatic** [comment]
I tota11y did, didn't I? 
Weird, but that's the kind of days I'm having... (Vir, B5)

Let's hope this one sticks!  @rtibbles

File: kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/datepicker.handlebars [hunk 1]
```diff
@@ -3,5 +3,5 @@
     <input type="text" id="start" class="form-control date-range" value={{ start_date }} name="start" />
     <span class="input-group-addon">→</span>
     <input type="text" id="end" class="form-control date-range" value={{ end_date }} name="end" />
-    <span class="input-group-addon btn-success setrange"><i class="glyphicon glyphicon-refresh"></i></span>
+    <span class="input-group-addon btn btn-success setrange" role="button"><span aria-hidden="true"><i class="glyphicon glyphicon-refresh"></i></span><span class="sr-only">Refresh</span></span>
```
Comment (rtibbles) [inline kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/datepicker.handlebars]: Translate!

**Comment by rtibbles** [comment]
Another missed translation tag!

**Comment by radinamatic** [comment]
I'm putting the i18n wiki pages in my in-depth-reading-list.

**Pull request merged**